decorator.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? "#{@nameable.correct_name[0..10]}..." : @nameable.correct_name
```

Comment:
- [ ] Kindly make the maximum length of the word 10 characters instead of 11 characters which is in this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=has%20a%20maximum%20of%2010%20characters.). As your code `@nameable.correct_name[0..10]` will return 11 characters.